IoTEdgeModbus/Modules/AzureIoTEdgeModbus/Instrumentation/MicrosoftExtensionsLog.cs

```diff
@@ -33,31 +33,31 @@ public void IoTHubConnectionOpened()
             this.LogInformation(MethodBase.GetCurrentMethod());
         }
         
-        [Event(2002, Description = "New desired properties received: {0}.")]
+        [Event(2002, Description = "New desired properties received: {desiredProperties}.")]
         public void DesiredPropertiesReceivedFromTwin(TwinCollection twinCollection)
         {
             this.LogInformation(MethodBase.GetCurrentMethod(), twinCollection);
         }
 
-        [Event(2003, Description = "New desired properties received: {0}.")]
+        [Event(2003, Description = "New desired properties received: {desiredProperties}.")]
         public void DesiredPropertiesReceivedFromFile(string desiredProperties)
         {
             this.LogInformation(MethodBase.GetCurrentMethod(), desiredProperties);
         }
 
-        [Event(2004, Description = "Attempting to retrieve desired configuration from provider: {0}.")]
+        [Event(2004, Description = "Attempting to retrieve desired configuration from provider: {providerType}.")]
         public void RetrieveDesiredConfigurationFrom(string configurationProviderTypeName)
         {
             this.LogInformation(MethodBase.GetCurrentMethod(), configurationProviderTypeName);
```

Comment:
Do we want to use reflection here? Consider using nameof(RetrieveDesiredConfigurationFrom) instead. All methods in the class seems to use reflection so in case we change we should change all of them.